### PR TITLE
Ethereal compat

### PR DESCRIPTION
**Pull request by Grossam**
Juste my two cents ;-)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,3 +2,29 @@
...
+* 2019-07-27 :
+  * Forked on https://git.fwhost.eu/Grossam/dmobs
+
+
+## Credits
+
+* dmobs_thunder.ogg : https://freesound.org/people/Josh74000MC/sounds/475094/
+* dmobs_wind.ogg : https://freesound.org/people/Pedaling%20Prince/sounds/338952/
+* dmobs_poison.ogg : https://freesound.org/people/qubodup/sounds/182789/
+* dmobs_wave.ogg : https://freesound.org/people/Kayyy/sounds/61011/
```
Comment (taikedz) [inline README.md:30]: You will need to add the explicit licenses to the `license.txt` for copyright compliance

`CC-0` for thunder, and `CC-BY 3.0` for the others.

**Comment by taikedz** [comment]
In principle, this looks fine - the changes seem sane, though as noted on the forum by Grossam, needs testing.

This also makes use of a newer version of `mobs-redo` with new spawning API, so worth marking which version it is meant to be compatible with (version number, or commit).

@Grossam - would you mind making a PR [here](https://github.com/minetest-mobs-mods/dmobs) too? I offered to move it there during #20 as I'm rather interested in mobs ;-)

**Comment by Grossam** [comment]
Second PR done. 

Should I precise the mobs_redo version in the readme ?
I'll deal with the licence when I find time ;-) (And maybe I modified some audio files that I shouldn't…)

**Comment by D00Med** [comment]
Thankyou for the contribution, but as far as I know this mod really doesn't work properly anymore. 
@taikedz would you like to become owner/contributor of this mod? At this point you are probably the most familiar with the mod.

**Comment by Grossam** [comment]
The mod is installed on one of my favorite servers (minetest.fwhost.eu:30000) where I spotted the bugs. I decided to fix them to contibute to the server and Minetest community (and to have a bit of fun ;-) ) and as I browsed the code it seemed to me it was kind of a WIP and needed completion. That's why I decided to work on it.

It's a bit the same with scifi nodes : it's full of nice ideas, but IMHO needed (and needs) some more work to be fully usable on a server (for instance you can't turn scifi blocks with screwdriver, some items have no recipe…)

The two mods are running on FWHmine, so they are tested the hard way, and you don't need to hurry about merging my commits : you may wait to see how things go, and even test my commits live. Indeed, you don't need to merge them at all : it won't stop me :-)

Regards ;-)

**Comment by taikedz** [comment]
@D00Med I'm happy to take this over on [the other repo][mmm] ; you might want to add a note to this one's readme to redirect, and Archive it so no new Issues can be raised against it anymore...

@Grossam I can't see any PR on [the other repo][mmm], sure it went to the right place ? :-) I'm totally interested in seeing progressive fixes on this mod, a merge would be welcome

[mmm]: https://github.com/minetest-mobs-mods/dmobs

**Comment by D00Med** [comment]
Ok, I will edit the readme

**Comment by Grossam** [comment]
There's an etheral-compat branch for the PR. But I can do it again.

**Comment by taikedz** [comment]
@Grossam Merged in the new repo location : https://github.com/minetest-mobs-mods/dmobs

**Pull request closed without merging**